src/components/save.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import Button from 'boundless-sdk/components/Button';
+import classNames from 'classnames';
+import enMessages from 'boundless-sdk/locale/en.js';
+import SaveIcon from 'material-ui/svg-icons/content/save';
+import {defineMessages, injectIntl, intlShape} from 'react-intl';
+import SaveDetailModal from './savedetailmodal';
+import {setMapConfig, saveMap} from '../reducers/actions'
+import { connectAdvanced } from 'react-redux'
+
+const messages = defineMessages({
+  savetext: {
+    id: 'savemap.savetext',
```

Comment:
minor, but normally I think the name of the component (so save) is used before the dot